src/applications/static-pages/subscribeAdditionalInfoEvents.js

```diff
@@ -0,0 +1,22 @@
+import { recordEventOnce } from 'platform/monitoring/record-event';
+
+export default function subscribeAdditionalInfoAnalytics() {
+  window.onload = () => {
```

Comment:
Does this need to be attached to the onload event? I don't think we were doing that before and the body element should always be there when the script runs, because they're deferred.

Reply:
I’d prefer not to but when I tested this the body wasn’t loaded when this script is run. I’ll look around at the startup code to see if can do this without an onload event.

Reply:
Meaning that document.body didn’t have any methods on it

Reply:
works without onload- updated